lib/beaker/host/windows/group.rb

```diff
@@ -14,6 +14,20 @@ def group_list
     end
   end
 
+  # using powershell commands as wmic is deprecated in windows 2025
+  def group_list_using_powershell
+    execute('cmd /c echo "" | powershell.exe "Get-LocalGroup | Select-Object -ExpandProperty Name"') do |result|
```

Comment:
should we be using the `powershell` dsl method to handle things like `-NoProfile`, etc?

```suggestion
    execute(powershell("Get-LocalGroup | Select-Object -ExpandProperty Name")) do |result|
```